src/foraging_gui/Dialogs.py

```diff
@@ -280,8 +278,8 @@ def _LaserColor(self,Numb):
             if no_calibration:
                 for laser_tag in self.laser_tags:
                     getattr(self, f"Laser{laser_tag}_power_{str(Numb)}").clear()
-                    self.MainWindow.WarningLabel.setText('No calibration for this protocol identified!')
-                    self.MainWindow.WarningLabel.setStyleSheet(self.MainWindow.default_warning_color)
```

Comment:
Previously we were setting the color of the texts to `default_warning_color` which is a purple color so its visible in the red-light room. I would get feedback from the RAs whether black or purple is preferable.

We can probably delete the default_warning_color if its not in use elsewhere

Reply:
Just added in ability to set color but will check with ra's